instrumentation/net/http/otelhttp/internal/semconv/httpconv.go

```diff
@@ -195,3 +195,143 @@ func (n newHTTPServer) ResponseTraceAttrs(resp ResponseTelemetry) []attribute.Ke
 func (n newHTTPServer) Route(route string) attribute.KeyValue {
 	return semconvNew.HTTPRoute(route)
 }
+
+type newHTTPClient struct{}
+
+// RequestTraceAttrs returns trace attributes for an HTTP request made by a client.
+func (n newHTTPClient) RequestTraceAttrs(req *http.Request) []attribute.KeyValue {
```

Comment:
Should those methods be tested?

Reply:
This is tested in the httpconv_test.go. Is there anywhere else I should add tests for this method?